WoWPro_Leveling/Wrath/Horde/WOTLK_Borean_Tundra.lua

```diff
@@ -317,9 +305,8 @@ C Kaw the Mammoth Destroyer |QID|11879|N|Find a Wooly Mammoth Bull and right-cli
 A The Honored Ancestors |QID|11605|N|From Elder Atkanok at the Coldrock Quarry.|M|54.69,35.69|
 A Reclaiming the Quarry |QID|11612|N|From Etaruk.|M|54.3,36.1|
 C Reclaiming the Quarry |QID|11612|N|Kill Beryl Treasure Hunters.|S|
-;QO
-N Elder Sagani |QID|11605|QO|2|N|Click the Elder here.|M|52.84,34.05|
-N Elder Kesuk |QID|11605|QO|1|N|Click the Elder here.|M|50.81,32.38|
+C Elder Sagani |QID|11605|QO|2|N|Click the Elder here.|M|52.84,34.05|
```

Comment:
For C steps, having the step text be a rewording of the quest objective is better.  Perhaps  "Identify Elder Sagani".

Reply:
Adding T tag to help with this.